src/ci/docker/host-x86_64/x86_64-gnu-integration/build-fuchsia.sh

```diff
@@ -2,50 +2,73 @@
 
 # Downloads and builds the Fuchsia operating system using a toolchain installed
 # in $RUST_INSTALL_DIR.
+#
+# You may run this script locally using Docker with the following command:
+#
+# $ src/ci/docker/run.sh x86_64-gnu-integration
 
 set -euf -o pipefail
 
-INTEGRATION_SHA=1011e3298775ee7cdf6f6dc73e808d6a86e33bd6
+# Set this variable to disable updating the Fuchsia checkout. This is useful for
```

Comment:
Set it to what? Can I just put `1` there or what?

Reply:
Any nonempty string will work, but adding 1 in the comment so it's less confusing.

Reply:
In a CI context you want to force the checkout to a known state before building. If you think it would be nicer, I could make this script accept a --keep-checkout flag to use inside the dev shell (or invert it and have the docker script pass an --overwrite flag or something). But with that you'd have to be careful not to run the full thing with run.sh or it will overwrite your local checkout, which is why I think it might be nicer to use the workflow of modifying this script directly.

Reply:
I mostly meant that in our CI context the builder should be starting from scratch without an existing checkout to keep IIUC. 

It could be an interesting change in the future, but as long as we document this, as you've done in the script and in the docs PR, then it should likely be fine for now.